libs/math/include/math/clustering/k_means.hpp

```diff
@@ -180,8 +180,8 @@ class KMeansImplementation
     }
 
     // initialise size of euclidean distance container
-    k_euclids_      = fetch::core::Vector<ArrayType>(n_clusters_);
-    empty_clusters_ = fetch::core::Vector<SizeType>(n_clusters_);
+    k_euclids_      = std::vector<ArrayType>(n_clusters_);
```

Comment:
There was a purposeful reason for using the `fetch::Vector` specifically the updated `operator[]` behaviour. What prompted this change?

Reply:
It's dead code - it's only used by k_means.hpp, which is not used anywhere. I'm not deleting k_means.hpp itself at Khan's request.